coderedcms/templates/coderedcms/blocks/accordion_block.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'coderedcms/blocks/base_block.html' %}
+
+{% load wagtailcore_tags coderedcms_tags %}
+
+{% block block_render %}
+
+{% generate_random_id as modal_id %}
+<a
+  href="#"
```

Comment:
I think accessibility standards might prefer this to be a button rather than an anchor, since it does not actually link to anywhere. Not sure if bootstrap4 supports this or not though. Just wanted to check.